tests/test_plugin_karma.py

```diff
@@ -79,6 +79,12 @@ def test_chained(self):
         self.assertEquals(self.bot.db[_format_key('foo')]['down'], 1)
         self.assertEquals(self.bot.db[_format_key('foo--')]['up'], 1)
 
+    def test_chained_brackets(self):
+        self.bot.dispatch(':root@localhost PRIVMSG #chan :(foo--)++')
```

Comment:
Note that this modifies the state in a way not documented by the test: `(foo` is given one negative karma.